external/kafka-0-10-sql/src/main/scala/org/apache/spark/sql/kafka010/KafkaWriter.scala

```diff
@@ -86,12 +86,10 @@ private[kafka010] object KafkaWriter extends Logging {
       topic: Option[String] = None): Unit = {
     val schema = queryExecution.analyzed.output
     validateQuery(queryExecution, kafkaParameters, topic)
-    SQLExecution.withNewExecutionId(sparkSession, queryExecution) {
```

Comment:
yea, because it's used in `KafkaProvider` which is run with `SaveIntoDataSourceCommand`, and all commands will be wrapped by `SQLExecution.withNewExecutionId`